snuba/datasets/cdc/cdcprocessors.py

```diff
@@ -0,0 +1,59 @@
+from snuba import processor
+from snuba.consumer import KAFKA_MESSAGE_OFFSET_KEY
+from snuba.processor import MessageProcessor
+
+
+class CdcProcessor(MessageProcessor):
+
+    def __init__(self, pg_table):
+        self.pg_table = pg_table
+
+    def _process_begin(self, offset):
+        pass
+
+    def _process_commit(self, offset):
+        pass
+
+    def _process_insert(self, offset, columnnames, columnvalues):
+        pass
+
+    def _process_update(self, offset, key, columnnames, columnvalues):
+        pass
+
+    def _process_delete(self, offset, key):
+        pass
+
+    def process_message(self, value, metadata):
+        assert isinstance(value, dict)
+        assert isinstance(metadata, dict)
+
+        offset = metadata[KAFKA_MESSAGE_OFFSET_KEY]
```

Comment:
We probably should be asserting somewhere that the partition is 0 (or 1? I forget how they're indexed) because if it's not, we're in a world of trouble.

Reply:
makes sense.